reference/7.1/Microsoft.PowerShell.Core/About/about_Parameters.md

```diff
@@ -21,11 +21,24 @@ parameters follow the command name and have the following form:
 -<parameter_name> <parameter_value>
 -<parameter_name>:<parameter_value>
 ```
-
-The name of the parameter is preceded by a hyphen (-), which signals to PowerShell that the word
-following the hyphen is a parameter name. The parameter name and value can be separated by a space
-or a colon character. Some parameters do not require or accept a parameter value. Other parameters
+<!--
+01234567890123456789012345678901234567890123456789012345678901234567890123456789
+-->
+The name of the parameter is preceded by a hyphen (-), which signals to
+PowerShell that the word following the hyphen is a parameter name.
+This effect can be switched off by escaping the hyphen (``Write-Output `-I``)
+or specifying the string as a parameter value (`Write-Output -InputObject:-I`).
+The special parameter `--` switches off recognising parameters for the rest
+of the command (`Write-Output -- -I`).
+The parameter name and value can be separated by a space or a colon character.
+Some parameters do not require or accept a parameter value. Other parameters
 require a value, but do not require the parameter name in the command.
+Named parameters passed to a command that does not expect them become positional
```

Comment:
I'm sorry if I'm missing something in the section, can you give an example of a command like this? It seems strance that a named param would become positional. I could be just misunderstanding something.

Reply:
```powershell
ECHO -A:B
```

Reply:
Hmm ok, what you mean. It's confusing because it reads like you are talking about named parameters that exist for the cmdlet and those would be interpreted as strings for some reason. Any possiblity of re wording to make it a little clearer?